apps/modernization-ui/src/apps/search/investigation/GeneralSearchFields.tsx

```diff
@@ -0,0 +1,237 @@
+                id="conditions-autocomplete"
+                name="conditions"
+                label="Conditions"
+                resolver={condtionsResolver}
+                onChange={handleChangeConditions}
+            />
+            {programAreaOptions?.length ? (
+                <Controller
+                    control={form.control}
+                    name="programAreas"
+                    render={({ field: { name } }) => (
+                        <MultiSelect
+                            data-testid="programArea"
+                            id="programArea"
+                            label="Program area"
+                            name={name}
+                            options={programAreaOptions?.map((p) => {
+                                return {
+                                    label: p.display ?? '',
+                                    name: p.display ?? '',
+                                    value: p.value ?? ''
+                                };
+                            })}
+                        />
+                    )}
+                />
+            ) : null}
+            <MultiSelect
+                data-testid="jurisdictions"
+                id="jurisdiction"
+                label="Jurisdiction"
+                name="jurisdiction"
+                onChange={() => 'void'}
+                options={[
+                    {
+                        label: 'Jurisdiction',
+                        name: 'coming soon',
+                        value: 'comingsoon'
+                    }
+                ]}
+            />
+            <Controller
+                control={form.control}
+                name="pregnancyStatus"
+                render={({ field: { name, value } }) => (
+                    <SelectInput
+                        data-testid="pregnancyStatus"
+                        name={name}
+                        id="pregnancyStatus"
+                        label="Pregnancy test"
+                        onChange={() => form.setValue('pregnancyStatus', value)}
+                        options={[
+                            { name: PregnancyStatus.Yes, value: PregnancyStatus.Yes },
+                            { name: PregnancyStatus.No, value: PregnancyStatus.No },
+                            { name: PregnancyStatus.Unknown, value: PregnancyStatus.Unknown }
+                        ]}
+                    />
+                )}
+            />
+            <Controller
+                control={form.control}
+                name="identification.type"
+                render={({ field: { value, name } }) => (
+                    <SelectInput
+                        name={name}
+                        defaultValue={asValue(value)}
+                        label="Event id type"
+                        dataTestid={name}
+                        htmlFor={name}
+                        options={Object.values(InvestigationEventIdType).map((event) => {
```

Comment:
suggestion: There were some `Selectable[]` [constants](https://github.com/CDCgov/NEDSS-Modernization/blob/main/apps/modernization-ui/src/apps/search/investigation/InvestigationFormTypes.ts) added for these lists.  Looks like I left out the Identification type though, could you add this list there?